demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -706,6 +737,16 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver,
           logLevel
         ),
       );
+
+      await this.createLogStream(configuration, 'create_browser_event_log_stream');
```

Comment:
Create 2 log streams after clicking on the Join button can create a noticable delay. Could we run both of them and wait for them at the end?

Reply:
A new revision is creating two log streams simulatenously. Thanks for the suggestion.

```
await Promise.all([
  this.createLogStream(configuration, 'create_log_stream'),
  this.createLogStream(configuration, 'create_browser_event_log_stream')
]);
```